src/models/task.class.js

```diff
@@ -0,0 +1,17 @@
+import { LEVELS } from "./levels.enum";
+
+export class Task {
```

Comment:
we dont name files like xxx.class.js. If it is a class or not, should be noticed by the definition of the component.

So It is not needed to call it.